src/glossary.md

```diff
@@ -1,23 +1,23 @@
-# Glossary
+# Glossary { #glossary }
```

Comment:
With https://github.com/google/mdbook-i18n-helpers/pull/107 by @dyoo just merged, we can now begin adding comments for the translators to our Markdown files!

```suggestion
<!-- i18n:comment Please keep { #glossary } untranslated. -->

# Glossary { #glossary }
```